plugins/pulp_docker/plugins/auth_util.py

```diff
@@ -93,7 +93,7 @@ def request_token(downloader, request, auth_header, repo_name):
     downloader.session.headers.pop('Authorization', None)
     report = downloader.download_one(token_request)
     if report.state == report.DOWNLOAD_FAILED:
```

Comment:
Maybe this would work better if we included the token data in the report.

Reply:
we do not really need this if report was successful.

Reply:
If we need a conditional later to know whether the download was successful, this conditional isn't really useful.